src/v/redpanda/admin/api-doc/config.def.json

```diff
@@ -1,6 +1,25 @@
-"set_log_level_response": {
+"get_log_level_response": {
```

Comment:
forgot that this was in this PR. gonna refactor the tests a bit and fold some later changes back to here.